android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -525,14 +527,47 @@ class StripeSdkModule(reactContext: ReactApplicationContext) : ReactContextBaseJ
     googlePayFragment?.createPaymentMethod(currencyCode, amount, promise)
   }
 
+  @ReactMethod
+  fun canAddCardToWallet(params: ReadableMap, promise: Promise) {
+    val last4 = getValOr(params, "cardLastFour", null) ?: run {
+      promise.resolve(createError("Failed", "You must provide cardLastFour"))
+      return
+    }
+
+    if (!PushProvisioningProxy.isNFCEnabled(reactApplicationContext)) {
+      promise.resolve(createCanAddCardResult(false, "UNSUPPORTED_DEVICE"))
+      return
+    }
```

Comment:
Is it a valid use case to tokenize a card into a device that doesn't have NFC, so that it can be used to pay online?

Reply:
Hm, I hadn't considered that use case. So you're saying to still return the `UNSUPPORTED_DEVICE` result, but attempt to get the google pay card token anyways?